gems/aws-sdk-cognitoidentity/lib/aws-sdk-cognitoidentity/customizations.rb

```diff
@@ -1,7 +1 @@
-# WARNING ABOUT GENERATED CODE
-#
-# This file is generated. See the contributing for info on making contributions:
-# https://github.com/aws/aws-sdk-ruby/blob/master/CONTRIBUTING.md
-#
-# WARNING ABOUT GENERATED CODE
-
+require 'aws-sdk-cognitoidentity/customizations/cognito_identity_credentials'
```

Comment:
Our travis tests run `rake build` but be sure that this file isn't overwritten (generate code warning has me nervous).

Reply:
Yeah - I was nervous about that.  It doesn't over write (note: other gem customization files do not have the generated src warning).